nemo/collections/common/tokenizers/huggingface/auto_tokenizer.py

```diff
@@ -15,7 +15,6 @@
 import re
 from typing import Optional
 
-from transformers import ALL_PRETRAINED_CONFIG_ARCHIVE_MAP
```

Comment:
Why do we remove ALL_PRETRAINED_CONFIG_ARCHIVE_MAP? If this doesn't return the list, do you know what does?